portfolio/src/main/webapp/pet-gallery.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-8">
+    <title>Pet Picture Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+      <div id="content">
+      <h1>Pet Image Gallery</h1>
+      <img src="/images/Dog2.JPG" width="225" height="300"></img>
```

Comment:
You could also specify the height and width in a css class and use the class here.